### DISCUSSION TITLE
针对当前版本完善建议

### DISCUSSION THREAD

**Discussion by basionven** [1Remote/1Remote#592] [Ideas]
针对当前的软件功能有几个建议，您看看是否考虑在后续版本中增加完善一下：
1、增加自定义分组功能，实现连接增删改操作，如增加自定义【广州RPD】或【FTP】，在该目录中新增或将其他目录的连接移至些目录；
2、继承：分组可设定相应的用户名与密码，新建的连接可根据实际需求选择是否继承组用户密码；
3、增加【扫描服务器】功能，可自定义扫描IP地址（开始段~结束段），将扫描到的服务器地址快速生成【连接】，用户仅需对重要信息（用户名与密码）填入即可实现连接；
4、信息简化：用户新建或进入已有连接信息页面，仅显示必要信息，需要显示更完整信息可通过相应指引或按钮功能切换实现；
5、快速ping:用户无需进入远程界面，直接返回ping结果，最好支持批量ping；
6、整合窗口：相关的连接直接在一个窗口用标签形式打开，也可快速切换回当前的服务列表；建议连接以标签形式打开时，当前【全部】那行不显示，直接把标签显示到窗口标题行，以使窗口最大化；状态栏设置为可隐藏/显示；
7、设置项：可设置一直沿用当前【分离窗口模式】或【整合窗口】模式；
8、地址名称即连接名称，用户默认直接填入远程IP时，名称默认以IP命名，也支持用户重新定义名称，方便用户快速输入；

**Comment by VShawn** [comment]
1、分组已经被现在的多标签系统取代，以后也不会加分组功能
2、不会做，类似的需求通过“标签分组筛选后，批量编辑修改功能”实现，具体可以翻看我们过去关于这一点的讨论。
3、请使用端口扫描工具，1Remote侧重于远程控制，并不准备成为一个大而全的网管工具、
5、考虑一下先，关键在于功能按钮放在哪才不突兀，结果怎么显示才好看。
6、也不会做，因为要实现6需要重构整个UI，没有那么多精力再去写一版UI了。
8、我考虑一下
